packages/sources/vue/src/components/overlays/VtmnModal/VtmnModal.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+import '@vtmn/css-modal/dist/index-with-vars.css';
+import { computed, defineComponent, PropType, reactive } from 'vue';
+import { VtmnButton } from '../../index';
+
+export default /*#__PURE__*/ defineComponent({
+  name: 'VtmnModal',
+  components: { VtmnButton },
+  inheritAttrs: false,
+  props: {
+    show: {
+      type: Boolean as PropType<Boolean>,
+      default: false,
+    },
+  },
+  emits: ['close'],
+  setup(props, { emit }) {
+    props = reactive(props);
+
+    const handleClose = () => {
+      return emit('close');
+    };
+
+    return {
+      classes: computed(() => ({
+        'vtmn-modal': true,
+      })),
+      handleClose,
+    };
+  },
+});
+</script>
+
+<template v-if="show">
```

Comment:
Warning here, you can't use v-if or anything on the <template> opening tag of a component !
The v-if="show" should be use on the div juste after this line.

Reply:
ok thanks :)